packages/MarkdownEditor-Tests.package/MarkdownCodeBlockTest.class/instance/testIndentation1Closes.st

```diff
@@ -0,0 +1,7 @@
+tests - extendability
+testIndentation1Closes
```

Comment:
Maybe these three or four tests can be refactored into one, as we discussed yesterday